ksqldb-common/src/main/java/io/confluent/ksql/util/timestamp/StringToTimestampParser.java

```diff
@@ -40,6 +41,25 @@ public StringToTimestampParser(final String pattern) {
     formatter = DateTimeFormatter.ofPattern(pattern, Locale.ROOT);
   }
 
+  /**
+   * Parse with a default time zone of {@code ZoneId#systemDefault}
+   *
+   * @see #parse(String, ZoneId)
+   */
+  public Timestamp parseToTimestamp(final String text) {
+    return parseToTimestamp(text, ZoneId.systemDefault());
+  }
+
+  /**
+   * @param text    the textual representation of the timestamp
+   * @param zoneId  the zoneId to use, if none present in {@code text}
+   *
+   * @return the millis since epoch that {@code text} represents
+   */
+  public Timestamp parseToTimestamp(final String text, final ZoneId zoneId) {
+    return Timestamp.from(parseZoned(text, zoneId).toInstant());
+  }
+
```

Comment:
Are there tests for these two functions? I don't see the `StringToTimestampParserTest`